src/contracts/Chef.sol

```diff
@@ -54,6 +54,13 @@ abstract contract Chef is Ownable {
         _lifetimeShareValue = _lifetimeShareValue.add(accumulatedMetricDividedByShares(accumulatedWithMetricPrecision));
     }
 
+    function getLifeTimeShareValueEstimate() public view virtual returns (uint256) {
```

Comment:
should we make `getLifetimeShareValue()` private to avoid a confusing public API?

Reply:
sure I can set it to internal - thinking of renaming it to getLastRecordedLifeTimeShareValue because essentially that's what it is - you need to call `setLifetimeShareValue` before you get the latest value - it's essentially a getter after a recent setter - but the value is only up to date after calling `setLifetimeShareValue`

Reply:
`getLifetimeShareValue` is now internal @gnomadic 
at @addyxvii let me know if you have thoughts about rename as well or is it making it to explicit

Reply:
I think it's a good name, don't think there's a better way to indicate that it's an imprecise value